### ISSUE TITLE
License

### ISSUE THREAD

**Issue by antagomir** [microbiome/OMA#231]
Reconsider the OMA license.

We have initially decided to have the NC clause for OMA license in because it is specifically used with books (see e.g. [Modern Statistics for Modern Biology](https://web.stanford.edu/class/bios221/book)).

On the other hand, our companion, the [OSCA book](https://bioconductor.org/books/release/OSCA), seems to be now with full CC-BY. This is also a requirement if we wish to release OMA through Bioconductor when it is mature (this year, hopefully).

I would like to reopen this discussion a bit and see if we should keep the current license or could/should we switch to CC-BY.

If anyone knows good literature discussing this question in the context of online books in more depth, link here..

**Comment by antagomir** [comment]
Hmm if this is indeed a requirement for Bioc books (?) then we should start to look at this actively.

**Comment by TuomasBorman** [comment]
Yes, we should proceed with the license change. I believe we have the necessary mandate from all authors, as all contributors are included in the OMA manuscript and have already agreed with this plan.

**Comment by TuomasBorman** [comment]
License changed to CC BY 4.0

https://github.com/microbiome/OMA/commit/404b3d213a46b957c6b7d86dab928d69ccfb4c84
